src/styles/notifications.less

```diff
@@ -263,4 +263,154 @@ ux-alert {
         background-color: @alert-dark-bg;
         border: 1px solid @alert-dark-border;
     }
+}
+
+/*
+ * Badge
+ */
+
+.ux-badge-container {
+    position: relative;
+
+    .ux-badge {
+        display: block;
+        position: absolute;
+        text-align: center;
+        background-color: #000;
+        color: #fff;
+        box-sizing: border-box;
+        padding: 0;
+        margin: 0;
+        text-transform: none !important;
+        font-family: @font-family !important;
+    }
+
+    &.ux-badge-small .ux-badge {
+        height: 12px;
+        min-width: 12px;
+        border-radius: 6px;
+        padding: 1px 4px;
+        font-size: 10px;
+        line-height: 0.95em;
+    }
+
+    &.ux-badge-medium .ux-badge {
+        height: 16px;
+        min-width: 16px;
+        border-radius: 8px;
+        padding: 2px 6px;
+        font-size: 12px;
+        line-height: 0.95em;
+    }
+
+    &.ux-badge-large .ux-badge {
+        height: 20px;
+        min-width: 20px;
+        border-radius: 10px;
+        padding: 2px 8px;
+        font-size: 16px;
+        line-height: 0.9em;
+    }
+
+    &.ux-badge-above,
+    &.ux-badge-top {
+        .ux-badge {
+            top: 0;
+        }
+
+        &.ux-badge-small .ux-badge {
+            top: -7px;
```

Comment:
I think you should be able to use `translateY` here as well, which would save having to set out a bunch of different hard-coded pixel values for each size variant.